src/Common/src/Interop/Windows/NtDll/Interop.RtlNtStatusToDosError.cs

```diff
@@ -10,8 +10,10 @@ internal partial class Interop
     internal partial class NtDll
     {
         // https://msdn.microsoft.com/en-us/library/windows/desktop/ms680600(v=vs.85).aspx
+		#pragma warning disable BCL0015 // Invalid Pinvoke call
```

Comment:
Suppressing warnings as per Jan's suggestion #29169

Reply:
You should not need this if you're only including the files in the non UAP builds. For example:



https://github.com/dotnet/corefx/blob/master/src/System.IO.FileSystem/src/System.IO.FileSystem.csproj#L9

https://github.com/dotnet/corefx/blob/master/src/System.IO.FileSystem/src/System.IO.FileSystem.csproj#L187-L216

Reply:
@JeremyKuhne 
Adding the condition `'$(UWPCompatible)' != 'true'` as per the suggestion throws an error `TypesMustExist : Type 'System.Data.SqlTypes.SqlFileStream' does not exist in the implementation but it does exist in the contract.`